elixir/Dockerfile

```diff
@@ -1,41 +1,29 @@
-ARG ALPINE_VERSION=3.18.2
-ARG OTP_VERSION=26.0.2
-ARG ELIXIR_VERSION=1.15.2
+ARG ALPINE_VERSION=3.18.4
+ARG OTP_VERSION=26.1.1
+ARG ELIXIR_VERSION=1.15.6
```

Comment:
I bumped our build image to these since it looks like [QEMU was fixed](https://gitlab.com/qemu-project/qemu/-/issues/1034) to build JIT crossplatform with `docker buildx`.

I also spun up a beefy runner so that we can build cross-platform images since we'll need to for the `relay` and `gateway`.

Reply:
See https://github.com/firezone/containers/blob/main/.github/workflows/build.yml#L17

Reply:
Ah, so we actually don't need `arm64` elixir images in production anymore, so that's been removed.